packages/common/aws.js

```diff
@@ -211,21 +239,25 @@ exports.s3ObjectExists = (params) =>
 * @param {Object} params - same params as https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#putObject-property
 * @returns {Promise} - promise of the object being put
 **/
-exports.s3PutObject = (params) => {
-  if (!params.ACL) params.ACL = 'private'; //eslint-disable-line no-param-reassign
-  return exports.s3().putObject(params).promise();
-};
+exports.s3PutObject = improveStackTrace(
+  (params) => exports.s3().putObject({
+    ACL: 'private',
+    ...params
+  }).promise()
```

Comment:
clever?  I suppose we should know the language but maybe a comment that this will just provide a default value for ACL?  take this or leave it.  I can see it biting me in the future.

Reply:
I've clearly become a big fan of the spread operator.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax